src/components/combo_box/combo_box_options_list/combo_box_options_list.tsx

```diff
@@ -349,17 +356,33 @@ export class EuiComboBoxOptionsList<T> extends Component<
             </p>
           );
         } else {
-          emptyStateContent = (
-            <div className="euiComboBoxOption__contentWrapper">
+          const highlightSearchValue = (text: string, searchValue: string) => {
+            const reg = new RegExp(/(\{searchValue})/, 'gi');
+            const parts = text.split(reg);
+            return (
               <p className="euiComboBoxOption__emptyStateText">
-                <EuiI18n
-                  token="euiComboBoxOptionsList.createCustomOption"
-                  default="Add {searchValue} as a custom option"
-                  values={{
-                    searchValue: <strong>{searchValue}</strong>,
-                  }}
-                />
+                {parts.map(part =>
+                  part.match(reg) ? <strong>{searchValue}</strong> : part
+                )}
```

Comment:
React warns about not having a `key` prop on the strong element, as it is part of an array.

```suggestion
                {parts.map((part, idx) =>
                  part.match(reg) ? (
                    <strong key={idx}>{searchValue}</strong>
                  ) : (
                    part
                  )
                )}
```